.rubocop.yml

```diff
@@ -121,3 +122,21 @@ Style/TrailingWhitespace:
 Style/VariableName:
   Severity: fatal
   Enabled: true
+
+Metrics/LineLength:
+  Description: 'Limit lines to 120 characters.'
```

Comment:
this is true - but with nested classes/modules etc and some sizeable variable names it is quite easy to be hitting the 80 character limit